src/doc/rustc/src/platform-support/vxworks.md

```diff
@@ -0,0 +1,51 @@
+# `*-wrs-vxworks`
+
+**Tier: 3**
+
+Targets for the VxWorks operating
+system.
+
+Target triplets available:
+
+- `x86_64-wrs-vxworks`
```

Comment:
Each of these targets must be directly linked to this file in the `platform-support.md` target file list.